examples/chip-tool/commands/example/ExampleCredentialIssuerCommands.h

```diff
@@ -44,6 +44,17 @@ class ExampleCredentialIssuerCommands : public CredentialIssuerCommands
 
         return CHIP_NO_ERROR;
     }
+
+    void SetupDeviceAttestationRevocationSetPath(const char * path) override
+    {
+        if (path)
```

Comment:
OK, so if path is null then we keep using whatever the previous path used to be?  Again, this all needs to be clearly documented in the API, and then we can have a discussion about what the right behavior is.